Products/CMFPlone/patterns/tinymce.py

```diff
@@ -177,22 +174,13 @@ def get_tiny_config(self):
 
         # add safe_html settings, which are used in backend for filtering:
         if not self.filter_settings.disable_filtering:
-            valid_tags = self.filter_settings.valid_tags
+            # we just enable the general html5 filtering set from TinyMCE
+            # (See https://www.tiny.cloud/docs/tinymce/latest/content-filtering/#schema)
+            # and add the "nasty_tags" as "invalid_elements"
+            tiny_config["schema"] = "html5"
+            # filter out invalid elements early
             nasty_tags = self.filter_settings.nasty_tags
-            custom_attributes = self.filter_settings.custom_attributes
-            safe_attributes = [safe_text(attr) for attr in html.defs.safe_attrs]
-            valid_attributes = safe_attributes + custom_attributes
-            # valid_elements : 'a[href|target=_blank],strong/b,div[align],br'
-            tiny_valid_elements = []
-            for tag in valid_tags:
-                tag_str = "{}[{}]".format(tag, "|".join(valid_attributes))
-                tiny_valid_elements.append(tag_str)
-            # We want to remove the nasty tag including the content in the
-            # backend, so TinyMCE should allow them here.
-            for tag in nasty_tags:
-                tag_str = "{}[{}]".format(tag, "|".join(valid_attributes))
-                tiny_valid_elements.append(tag_str)
-            tiny_config["valid_elements"] = ",".join(tiny_valid_elements)
```

Comment:
Are you sure we don't need this anymore?
We need to make sure, that what ever is set in the HTML filter control panal is not in conflict with what TinyMCE filtering does. Is that's the case I'm fine with the simplification.

Reply:
You're right. Only relying on the "html5" schema doesn't fit our `safe_html` transformation. I've updated this to fix our test scenarios again.